build_rules.gradle

```diff
@@ -1253,6 +1254,125 @@ ext.applyGrpcNature = {
 
 /*************************************************************************************************/
 
+ext.applyPortabilityNature = {
+  println "applyPortabilityNature with " + (it ? "$it" : "default configuration") + " for project $project.name"
+  applyJavaNature(enableFindbugs: false, enableErrorProne: false, shadowClosure: {
```

Comment:
We've been working hard to fix ErrorProne warnings per-module with the eventual goal of making ErrorProne violations produce errors by default. How about hoisting the `enableErrorProne`configuration as a parameter to `applyPortabilityNature()`. Then make sure to retain the configured state for migrated projects.

Reply:
If that's the case, can we make `failOnWarning: true`? Eventually we want to remove the switch altogether and make failOnWarning default; if these projects are only generated code then I wouldn't expect to get any violations anyway. ErrorProne is configured to ignore generated code.

Reply:
We have no control over the generated code so it seems silly to force it to a standard such as error prone/findbugs

Reply:
I agree that the value of static analysis on a project of all generated code is dubious. But there is value in getting the entire codebase opted-in to static analysis, and our build logic will be simpler if there's just one set of configuration.

Since ErrorProne passes, let's enable that. My understanding is that there is effort to remove FindBugs, so don't worry about that for now.

Reply:
Done